layouts/partials/postlist.html

```diff
@@ -0,0 +1,6 @@
+<div style="font-size: 15px;">
```

Comment:
I think we should avoid direct styling if we can.

I would use same element/class here as we have in single post:
```
<p class="edition>
```

Then, in CSS, we can do:
```css
p.edition {
    /* styles that are common, like bold and italic, maybe font size */
}
.post .matter p.edition {
    /* styles specific to edition on posts list page */
}
main > p.edition {
    /* styles specific to edition on single post, like right-alignment */
}
```

Reply:
I was looking for a solution where I didn't touch the css file, but now that you recommend it, I'll go ahead and do it